tools/db_bench_tool.cc

```diff
@@ -2085,6 +2086,9 @@ class Benchmark {
     InstrumentedCondVar cv_;
     bool no_auto_recovery_;
     bool recovery_complete_;
+#endif  // ROCKSDB_LITE
+    bool WaitForRecovery(uint64_t /*abs_time_us*/) { return true; }
```

Comment:
#else ?

Reply:
#else is not needed here, basically from LITE's view, the class is like this:
```
  class ErrorHandlerListener : public EventListener {
   public:	   
    bool WaitForRecovery(uint64_t /*abs_time_us*/) { return true; }
    void EnableAutoRecovery(bool /*enable*/) {}
  };
```

Reply:
It's a compilation error in non-LITE mode. See Travis results.

Reply:
yes you are right

Reply:
Right. Good catch.

Reply:
Actually, missing a `public` here.

Reply:
nvm, I was wrong.